pagopa/io-app
# fix(IT Wallet): [SIW-1726] Fix accessibility issues within credential details screen

## Short description
This pull request addresses multiple accessibility issues on the credential details screen, ensuring a better experience for screen reader (SR) users.

## List of changes proposed in this pull request
- Updated `io-app-design-system` to `2.1.2`
- Added required locales
- Added accessibilty props to the trustmark QR Code image in the trustmark bottom sheet
- Added accessibilty props to the credential's skeumorphic card
- Fixed accessibilty for image claims, which where not announced with the correct role
- Added feedback announcement for the skeumorphic card's front/back toggle button
- Added feedback announcement for the hide/show claims data toggle button
- Added accessibility label to the fiscal code barcode

## How to test
With screen reader enabled, navigate to the MDL credential details screen and verify:
- The credential card has appropriate alternative text and is announced clearly by the SR.
- The front/back toggle button on the credential card provides feedback on its action.
- The hide/show claims data toggle button provides feedback on its state change.
- The Trustmark QR Code is announced correctly as an image.
- The profile image is recognized as an image and announced appropriately.

Navigate to the TS credential details screen and verify
- The fiscal code barcode is announced correctly as na image
